lib/about_yml/schema.json

```diff
@@ -17,48 +17,69 @@
             "description": "The type of content in the repo",
             "enum": ["app", "docs", "policy"]
         },
-        "owner_type": {
-            "type": "string",
-            "description": "Describes whether a project team, working group/guild, etc. owns the repo",
-            "enum": ["guild", "working-group", "project"]
+        "owner": {
+          "organization": {
+              "type": "string",
+              "description": "Short name for the organization that owns the project"
+          },
+          "team": {
+            "type": {
+                "type": "string",
+                "description": "Describes whether a project team, working group/guild, etc. owns the repo",
+                "enum": ["guild", "working-group", "internal-project", "client-project"]
+            },
+            "id": {
```

Comment:
I didn't notice "id" getting copied in when we were chatting about it in real-time. Mistake? Seems out of place in "owner".